data/array/lemmas.lean

```diff
@@ -186,3 +192,35 @@ end array
 
 instance (α) [decidable_eq α] : decidable_eq (buffer α) :=
 by tactic.mk_dec_eq_instance
+
+local prefix `♯`:0 := cast (by simp)
+
+namespace array
+
+variables {n : ℕ} (α : Type u)
+
+open function
+
+def array_equiv_vector : flip vector n α ≃ array n α :=
```

Comment:
This definition need not use `flip vector`. Moreover, I believe it's proven in `data.equiv`

Reply:
👍